test/functional/cmdLineTests/shareClassTests/SCCommandLineOptionTests/nameOption10.bat

```diff
@@ -37,7 +37,7 @@ set NAME=*%u
 if exist %TESTSCRIPT%.out (
     %1/java SimpleGrep "JVMSHRC147E Character * not valid for cache name" %TESTSCRIPT%.out 1
 )else (
-    echso %TESTSCRIPT%: TEST FAILED: No file created
+    echo %TESTSCRIPT%: TEST FAILED: No file created

```

Comment:
This change appears in a number of places and is really an odd thing. I'm treating it as a typo. But...

Reply:
I think you're right. If the test failed, it would just not be reported in the way I believe it was intended to be reported.